io/pci/dlpar.py

```diff
@@ -153,10 +153,13 @@ def install_packages(self):
         Install required packages
         '''
         smm = SoftwareManager()
+        packages = ['ksh', 'src', 'rsct.basic', 'rsct.core.utils',
+                    'rsct.core', 'DynamicRM']
         detected_distro = distro.detect()
         self.log.info("Test is running on: %s", detected_distro.name)
-        if not smm.check_installed("ksh") and not smm.install("ksh"):
-            self.cancel('ksh is needed for the test to be run')
+        for pkg in packages:
+            if not smm.check_installed(pkg) and not smm.install(pkg):
+                self.cancel('%s is needed for the test to be run' % pkg)
         if detected_distro.name == "Ubuntu":
```

Comment:
just a one minor comment can this check also clubbed with `https://github.com/avocado-framework-tests/avocado-misc-tests/pull/1094/files#diff-f5d7c870f9fca9cbe842de08c146d2e8R160`